utils/debug-grammar.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Add env variable `VSCODE_TEXTMATE_DEBUG=true` to run script with extra debugging info
+ * Example: `VSCODE_TEXTMATE_DEBUG=true node utils/debug-grammar.js example.apib`
+ */
+
+const fs = require('fs');
+const path = require('path');
+const vsctm = require('vscode-textmate');
+const oniguruma = require('vscode-oniguruma');
+
+const FILE_PATH = process.argv[2];
+
+const fileContents = fs.readFileSync(FILE_PATH, { encoding: 'utf-8' });
+const fileLines = fileContents.split(/\r\n|\r|\n/);
+
+const scopeToGrammar = {
+  'text.html.markdown.source.gfm.apib': getPath('../syntax/APIBlueprint.tmLanguage'),
+  'text.html.markdown.source.gfm.mson': getPath('../syntax/MSON.tmLanguage'),
+  'source.js': getPath('../syntax/JavaScript.tmLanguage'),
```

Comment:
So, this files are used in this util only. Should we keep them in syntax folder than? For the one who does not know about this util it may look like the plugin defines that syntaxes highlighting. But it's not, as I see in package.json 🤔 

Wouldn't it be better if we store JS and HTML tmLanguage files in the utils folder too?

Reply:
You're right, since these files are only for debug purposes (I believe, VS Code has them somewhere internally when running installed extension) we can move them into utils, for sure.